Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -496,5 +496,67 @@ protected override void ProcessRecord()
         }
     }
 
-    
+    /// <summary>
+    /// <para type="synopsis">
+    /// Requests that a running BigQuery Job be canceled.
+    /// </para>
+    /// <para type="description">
+    /// Requests that a job be canceled. This call will return immediately, and the client 
+    /// is responible for polling for job status. Canceled jobs may still incur costs. 
+    /// This cmdlet returns a Job object if successful.
+    /// </para>
+    /// <example>
+    ///   <code>
+    /// PS C:\> $job = Start-BqJob -Query "SELECT * FROM book_data.classics"
+    /// PS C:\> $job = $job | Stop-BqJob | Get-BqJob
+    /// PS C:\> $job.Status.State
```

Comment:
I don't think you need to add this line. Just add in the comment that the $job.Status.State should be "Done".